content/docs/getting-started/edit-content.md

```diff
@@ -2,33 +2,19 @@
 title: Editing Content
 prev: /docs/getting-started/cms-set-up
 next: /docs/getting-started/backends
+last_edited: '2020-08-18T08:13:26.090Z'
 ---
-
 The purpose of a CMS is to allow editors to change content. [Forms](/docs/plugins/forms) are a fundamental part of this as they define the editing interface for your content. In this step, we will **create and register a form to edit data** rendered on this page.
 
-## Create & Register a Form
-
-We will use the `useForm` hook to [create the form](/docs/plugins/forms#creating-forms).
-
-Calling `useForm` won't automatically make our form appear in the sidebar. There are multiple ways to use forms with Tina, and the sidebar is merely one option.
-
-Although it doesn't automatically appear in the sidebar, the form returned by `useForm` is pre-configured to work as a sidebar plugin. To add this form to the sidebar, all we have to do is pass it into the `usePlugin` hook.
-
+> _Fun Fact:_ Forms are a type of [plugin](/docs/plugins) in Tina. There are multiple ways to use forms with Tina, and the sidebar is merely one option.
 
-> _Fun Fact:_ Forms are a type of [plugin](/docs/plugins) in Tina.
+## Import the `useForm` and `usePlugin` hooks
 
-### The Steps
```

Comment:
I don't like that these steps have been removed. The reason the form configuration details are at the end of the page is that it is _extra_ information about _how things work_ that is not required to get through the tutorial. 

In general, I have issues with all of the changes on this page as they make the steps feel much longer. If there is additional explanation needed, we could add it as a paragraph below the single code example to further explain the steps.